### ISSUE TITLE
Color palette; borders between color blocks off

### ISSUE THREAD

**Issue by clkantner** [microsoft/pxt#9264]
In the dialogue, there's a black pixel border around each color. This is very helpful, especially for color accessibility. But the one small issue is the border between colors in the middle are thicker than on the outside. If there's a way to have the borders around each color the same, it will be much cleaner! 

<img width="567" alt="Screenshot 2022-12-08 at 2 50 28 PM" src="https://user-images.githubusercontent.com/106267801/206553528-03665a1d-52df-4da6-90bc-69ebd7f95556.png">
<img width="381" alt="Screenshot 2022-12-08 at 2 50 42 PM" src="https://user-images.githubusercontent.com/106267801/206553635-fcb0cf87-a044-4dbf-b5ab-fe5454d07430.png">
